_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -124,29 +120,21 @@ def __init__(self):
             "nation": {"pop": None},
             "jhu_uid": {"fips": None},
         }
-        self.geo_lists = {
+        self._geo_sets = {
             geo: None for geo in ["zip", "fips", "hrr", "state_id", "state_code",
-                                  "state_name", "hhs", "msa"]
+                                  "state_name", "hhs", "msa", "nation"]
         }
-        self.geo_lists["nation"] = {"us"}
-
-    # Utility functions
-    def _load_crosswalk(self, from_code, to_code):
-        """Load the crosswalk from from_code -> to_code."""
-        assert from_code in self.crosswalk_filepaths, \
-            f"No crosswalk files for {from_code}; try {'; '.join(self.crosswalk_filepaths.keys())}"
-        assert to_code in self.crosswalk_filepaths[from_code], \
-            f"No crosswalk file from {from_code} to {to_code}; try " \
-            f"{'; '.join(self.crosswalk_filepaths[from_code].keys())}"
-
-        if self.crosswalks[from_code][to_code] is None:
-            self.crosswalks[from_code][to_code] = self._load_crosswalk_from_file(from_code, to_code)
-        return self.crosswalks[from_code][to_code]
-
-    def _load_crosswalk_from_file(self, from_code, to_code):
```

Comment:
While we're refactoring this utility for clarity, I think we can get rid of most of the logic in this function by writing 
```
def _load_crosswalk_from_file(self, from_code, to_code, data_path):
    stream = pkg_resources.resource_stream(__name__, data_path)
    dtype = {
        from_code: str,
        to_code: str,
        "fips": str,
        "zip": str,
        "hrr": str,
        "hhs": str,
        "msa": str,
        "state_code": str,
        "state_id": str,
        "state_name": str,
        "pop": int,
        "weight": float
    }
    usecols = [from_code, "pop"] if to_code == "pop" else None
    return pd.read_csv(stream, dtype=dtype, usecols=usecols)
```
since `pd.read_csv` will just ignore irrelevant columns and there's no clash in the dtypes otherwise.